samples/core/guide/autograph.ipynb

```diff
@@ -377,7 +379,7 @@
         "  result = fizzbuzz(num)\n",
         "  with tf.Session() as sess:\n",
         "    for n in range(10,16):\n",
```

Comment:
Not related to this PR, but I feel that we should avoid examples that place control flow outside sess.run - in a way, AutoGraph was made so that one doesn't have to use that, which is a bit of an antipattern. For the point above, that converted code works like a regular op, I think we can use anything, like for instance taking the output and passing it through some other op, so we don't strictly need the fizzbuzz example for that. LMK what you think.

Reply:
I see your point.

Here I'm trying to show that FizzBuzz is working, without getting into too-complex of an example.

This would be a step in the right direction:

```
@autograph.convert()
def fizzbuzz(num):
  if num % 3 == 0 and num % 5 == 0:
      return 'FizzBuzz'
  elif num % 3 == 0:
      return 'Fizz'
  elif num % 5 == 0:
      return 'Buzz'
  else:
      return tf.as_string(num)
    
@autograph.convert()
def fizzbuzz_loop(start, stop):
  for n in range(start, stop):
    print(fizzbuzz(n))
  return tf.constant(stop)

with tf.Graph().as_default():
  # The result works like a regular op: takes tensors in, returns tensors.
  # You can inspect the graph using tf.get_default_graph().as_graph_def()
  with tf.Session() as sess:
    sess.run(fizzbuzz_loop(10,16))
```

But `print` is currently not working...